package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "jojen",
-  "version": "0.1.0-alpha.0",
+  "version": "0.1.0",
   "description": "Expressive validator for Node and the browser, with a small footprint and awesome performance",
-  "main": "lib/index.js",
+  "main": "dist/index",
   "scripts": {
+    "debug": "node debug mocha -r test/_setup.js --compilers js:babel-register && eslint lib",
     "test": "mocha -r test/_setup.js --compilers js:babel-register && eslint lib",
-    "sizeBrowser": "browserify lib/index.js -t babelify --presets es2015 | ccjs --compilation_level=ADVANCED_OPTIMIZATIONS | gzip | wc -c",
-    "prepublish": "babel lib --out-dir dist"
+    "bundle": "browserify lib/index.js -t babelify --presets es2015 | ccjs --compilation_level=ADVANCED_OPTIMIZATIONS > jojen.min.js",
+    "prepublish": "babel lib --out-dir dist && (type casdcjs &> /dev/null && npm run bundle || echo 'closure compiler not installed, not bundling for browser')"
```

Comment:
I would just put the closure compiler as an optional dependency and leave it at that.